### ISSUE TITLE
Moderation queue for shared pipes

### ISSUE THREAD

**Issue by anewuser** [pipes-digital/pipes#111]
Spammers are using the `Make Public` feature to advertise their links. Example: https://www.pipes.digital/pipe/aNgE7aqJ . There might be other forms of abuse too.

It's maybe a better idea for you to review pipes before they show up on the [Shared Pipes page](https://www.pipes.digital/pipes).

**Comment by onli** [comment]
What a crazy spam target! Do you see any instance where that link becomes a real link?

**Comment by anewuser** [comment]
No, but this one made it easier for people to copy the link: https://www.pipes.digital/pipe/EOm1b2OL

>polycarbonate sheet, polytop We are the sole official importer. Production and distribution of products, polycarbonate sheets, complete polycarbonate sheet, abbreviated as , poly sheet, polycarbonate sheet price
